aten/src/ATen/core/Reduction.h

```diff
@@ -9,6 +9,7 @@ enum Reduction {
   None,             // Do not reduce
   Mean,             // (Possibly weighted) mean of losses
   Sum,              // Sum losses
+  BatchMean,        // Mean over batches. = Sum / batchsize
```

Comment:
Can't you just hack this up in python? It should only be a workaround for kl_div and doesn't make sense for other losses.